src/main/java/backend/model/challenges/Challenge.kt

```diff
@@ -139,7 +138,7 @@ class Challenge : BasicEntity, Billable {
         this.team = team
         this.amount = amount
         this.description = description
-        this.contract = Media(DOCUMENT)
+        this.contract = null //TODO: how to handle contracts in future?
```

Comment:
I'd say revert that change and keep type document? We want to handle it in the same way, we can still upload assets to cloudinary even if they are not images or videos?

Reply:
We'll leave this for now, until we decided how to handle offline sponsors with this system now at all

Reply:
https://github.com/BreakOutEvent/Documentation/issues/137